web/config/setupTests.js

```diff
@@ -1,29 +0,0 @@
-import Enzyme from "enzyme";
```

Comment:
There are a couple of test set up files that I deleted. It doesn't look like they're used.